funcs/semver.go

```diff
@@ -0,0 +1,40 @@
+package funcs
+
+import (
+	"context"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+// CreateSemverFuncs -
+func CreateSemverFuncs(ctx context.Context) map[string]interface{} {
+	ns := &SemverFuncs{ctx}
+	return map[string]interface{}{
+		"semver": func() interface{} { return ns },
+	}
+}
+
+// SemverFuncs -
+type SemverFuncs struct {
+	ctx context.Context
+}
+
+// Semver -
+func (SemverFuncs) Semver(version string) (*semver.Version, error) {
+	return semver.NewVersion(version)
+}
+
+// MatchConstraint -
+func (SemverFuncs) MatchConstraint(constraint, in string) (bool, error) {
```

Comment:
Can we name this `CheckConstraint` instead so it's closer to the library's naming?

```suggestion
func (SemverFuncs) CheckConstraint(constraint, in string) (bool, error) {
```